MantidPlot/src/Mantid/InstrumentWidget/InstrumentWindowRenderTab.cpp

```diff
@@ -567,6 +567,12 @@ QMenu* InstrumentWindowRenderTab::createPeaksMenu()
   connect(signalMapper, SIGNAL(mapped(int)), m_instrWindow, SLOT(setPeakLabelPrecision(int)));
   menu->addMenu(setPrecision);
 
+  QAction *indicateRelativeIntensity = new QAction("Indicate relative intensity", this);
+  indicateRelativeIntensity->setCheckable(true);
+  indicateRelativeIntensity->setChecked(settings.value("IndicateRelativeIntensities", false).toBool());
```

Comment:
It seems this is not enough to restore the menu state. I'll try to find out if the menu is modified elsewhere.